getting-started-with-api/java-sdk/src/main/java/DownloadPseudo.java

```diff
@@ -0,0 +1,71 @@
+import com.smartling.api.files.v2.FilesApi;
+import com.smartling.api.files.v2.FilesApiFactory;
+import com.smartling.api.files.v2.pto.DownloadTranslationPTO;
+import com.smartling.api.files.v2.pto.RetrievalType;
+import com.smartling.api.v2.authentication.AuthenticationApi;
+import com.smartling.api.v2.authentication.AuthenticationApiFactory;
+import com.smartling.api.v2.client.ClientFactory;
+import com.smartling.api.v2.client.DefaultClientConfiguration;
+import com.smartling.api.v2.client.auth.Authenticator;
+import com.smartling.api.v2.client.auth.BearerAuthSecretFilter;
+
+import com.smartling.api.v2.client.exception.RestApiRuntimeException;
+import org.apache.commons.io.IOUtils;
+import java.io.InputStream;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class DownloadPseudo
+{
+    public static void main( String[] args ) throws Exception
+    {
+        // Read command-line arguments
+        if (args.length != 2) {
+            System.out.println("Usage: mvn -X exec:java -Dexec.mainClass=\"DownloadPseudo\" -Dexec.args=\"<fileUri> <localeId>\"");
```

Comment:
The command looks too long. Is it common to create a maven goal?

Reply:
Here's the example how to configure multiple main classes inside Maven config, so one don't need to specify main class, just configuration name: https://stackoverflow.com/a/51505284

Reply:
Thanks, will create maven goals like that and simplify the usage instructions.